doc/src/LeastSquareLine.mdoc

```diff
@@ -0,0 +1,125 @@
+Detailed Description
```

Comment:
Why do you copy the source code and all the comments here again ? 
Doxygen should be able to extract all this directly from source.